developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -209,3 +357,50 @@ File properties can be requested by adding a :code:`<d:prop/>` element to the re
 When listing favorites, the request will find all favorites in the folder recursively, all favorites for a user can be found by sending the request to :code:`remote.php/dav/files/user`
 
 .. _rfc4918: https://tools.ietf.org/html/rfc4918
+
+
+Special Headers
+---------------
+
+Request Headers
+^^^^^^^^^^^^^^^
+
+You can set some special headers that Nextcloud will interpret.
+
++-----------------+-----------------------------------------------------------------+------------------------------------------+
+|     Header      |                           Description                           |                 Example                  |
++=================+=================================================================+==========================================+
+| X-OC-MTime      | | Allow to specify a modification time.                         | ``1675789581``                           |
+|                 | | The response will contain the header ``X-OC-MTime: accepted`` |                                          |
+|                 | | if the mtime was accepted.                                    |                                          |
++-----------------+-----------------------------------------------------------------+------------------------------------------+
+| X-OC-CTime      | | Allow to specify a creation time.                             | ``1675789581``                           |
+|                 | | The response will contain the header ``X-OC-CTime: accepted`` |                                          |
+|                 | | if the mtime was accepted.                                    |                                          |
++-----------------+-----------------------------------------------------------------+------------------------------------------+
+| OC-Checksum     | | A checksum that will be stored in the DB.                     | ``md5:04c36b75222cd9fd47f2607333029106`` |
+|                 | | The server will not do any sort of  validation.               |                                          |
```

Comment:
https://github.com/nextcloud/desktop/blob/613357af70217f56ee697f6be425200a9f680067/doc/architecture.rst?plain=1#L210

Reply:
Hm ok, then the doc in the desktop client has to be wrong.